inline_Ob.cc

```diff
@@ -163,6 +163,7 @@ namespace Chroma
 	multi1d<LatticeColorMatrix> u;
 	u = TheNamedObjMap::Instance().getData< multi1d<LatticeColorMatrix> >(params.named_obj.gauge_id);
 
+	Wloop(xml_out, "GMF_O_b", u);
```

Comment:
The chroma function for 1*1, 1*2, 1*3 wilson loop.
The output are:
 <Wilson_loop_11>
            <all_av>0.99522569342592</all_av>
            <space_av>0.9952352709262</space_av>
            <time_av>0.995216115925639</time_av>
            <plane_01_plaq>0.995233099721999</plane_01_plaq>
            <plane_02_plaq>0.995222970514034</plane_02_plaq>
            <plane_12_plaq>0.995249742542568</plane_12_plaq>
            <plane_03_plaq>0.995221308703021</plane_03_plaq>
            <plane_13_plaq>0.995238434360685</plane_13_plaq>
            <plane_23_plaq>0.995188604713212</plane_23_plaq>
          </Wilson_loop_11>
          <Wilson_loop_12>
            <all_av>0.982568947036192</all_av>
            <space_av>0.982606241006113</space_av>
            <time_av>0.982531653066271</time_av>
            <plane_01_plaq>0.982599893797826</plane_01_plaq>
            <plane_02_plaq>0.982561716761413</plane_02_plaq>
            <plane_12_plaq>0.982657112459099</plane_12_plaq>
            <plane_03_plaq>0.982548033474641</plane_03_plaq>
            <plane_13_plaq>0.982621222778429</plane_13_plaq>
            <plane_23_plaq>0.982425702945742</plane_23_plaq>
          </Wilson_loop_12>
          <Wilson_loop_13>
            <all_av>0.964970896246458</all_av>
            <space_av>0.96505283298035</space_av>
            <time_av>0.964888959512565</time_av>
            <plane_01_plaq>0.965042810136821</plane_01_plaq>
            <plane_02_plaq>0.96496109201456</plane_02_plaq>
            <plane_12_plaq>0.965154596789669</plane_12_plaq>
            <plane_03_plaq>0.964912907242759</plane_03_plaq>
            <plane_13_plaq>0.965086551487113</plane_13_plaq>
            <plane_23_plaq>0.964667419807824</plane_23_plaq>
          </Wilson_loop_13>